iac/get_oncall.tf

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 The Sigstore Authors
+# SPDX-License-Identifier: Apache-2.0
+
+resource "google_service_account" "slack_slash_pg_sa" {
+  project = var.project_id
+
+  account_id   = "${var.name}-get-oncall"
+  display_name = "GetOncall - Slack slash Pagerduty"
+  description  = "Dedicated service account for the Slack slash Pagerduty service."
+}
+
+data "google_secret_manager_secret" "slack_secret" {
+  secret_id = "SLACK_SECRET"
+}
+
+resource "google_secret_manager_secret_iam_binding" "slack_api_secret_binding" {
+  secret_id = data.google_secret_manager_secret.slack_secret.id
+  role      = "roles/secretmanager.secretAccessor"
+  members = [
+    "serviceAccount:${google_service_account.slack_slash_pg_sa.email}",
+  ]
+}
+
+module "slack_slash_pg_service" {
+  source  = "chainguard-dev/common/infra//modules/regional-go-service"
```

Comment:
will this work? I doubt that source code is public?

Reply:
it is :) : https://github.com/chainguard-dev/terraform-infra-common

it a working that we have being doing to encapsulate and make it easier to use